janitor/functions.py

```diff
@@ -920,39 +921,57 @@ def deconcatenate_column(
 
     This method does not mutate the original DataFrame.
 
+    When `preserve_position=True`, `new_column_names` replaces original
+     `column_name` and preserves the column order (`column_name` is dropped);
+    otherwise, `new_column_names` is appended to the right of the dataframe
+
     Functional usage example:
 
     .. code-block:: python
 
-        df = deconcatenate_columns(df,
-                                   column_name='id',
-                                   new_column_names=['col1', 'col2'],
-                                   sep='-')
+        df = deconcatenate_column(
+                df, column_name='id', new_column_names=['col1', 'col2'],
+                sep='-', preserve_position=True
+        )
 
     Method chaining example:
 
     .. code-block:: python
 
         df = (pd.DataFrame(...).
-              deconcatenate_columns(column_name='id',
-                                    new_column_names=['col1', 'col2'],
-                                    sep='-'))
+                deconcatenate_column(
+                    column_name='id', new_column_names=['col1', 'col2'],
+                    sep='-', preserve_position=True
+                ))
+        # When `preserve_position=True`
```

Comment:
I think this can be written not as an in-line comment, but as part of the docstring officially. 

Can you put this above the examples instead? Once done, then see if rendering the docs makes the docs look correct?